docs/server/generate_schema_documentation.py

```diff
@@ -0,0 +1,156 @@
+""" Script to render transaction schema into .rst document """
+
+import os.path
+
+from bigchaindb.common.schema import TX_SCHEMA_ORDERED as TX
+
+
+TPL_PROP = """\
+%(title)s
+%(underline)s
+
+**type:** %(type)s
+
+%(description)s
+"""
+
+
+TPL_DOC = """\
```

Comment:
Should probably put a rst comment at the start of this document to let others know that it's deliberately auto generated from the schemas.